### PR TITLE
ci: minor cleanup of E2E job

### PR DESCRIPTION
**Pull request by nathan-weinberg**
To align with implementation in other library repos

Note `ilab system info` is removed as that is now part of the testing script itself

### CODE REVIEW

File: .github/workflows/e2e-nvidia-t4-x1.yml [hunk 1]
```diff
@@ -64,29 +63,30 @@ jobs:
...
       # for debugging
-      - name: Print Environment State
+      - name: Print environment state
         run: |
           echo "Current Working Directory: $PWD"
           echo "Files in Local Directory:"
           ls -l
-      - name: Checkout
+
+      - name: Checkout instructlab/instructlab
```
Comment (JamesKunstle) [inline .github/workflows/e2e-nvidia-t4-x1.yml:72]: do we want to be checking out the current main branch of instructlab if we're testing training? I assume we might prefer to use the most recently published version instead.

Comment (nathan-weinberg) [reply .github/workflows/e2e-nvidia-t4-x1.yml:72]: Not something that should be changed as part of this PR

Comment (JamesKunstle) [reply .github/workflows/e2e-nvidia-t4-x1.yml:72]: okay

**Review by JamesKunstle — APPROVED** [review]
tests still pass and it's an in-place rename

**Review by danmcp — APPROVED** [review]

**Pull request merged**